test/Factories.hs

```diff
@@ -27,13 +21,28 @@ buildSite = Site
     , siteLanguage = "en-us"
     }
 
+createFreePlan :: DB (Entity Plan)
+createFreePlan = upsert Plan
```

Comment:
I think the docs are wrong. `repsert` requires a `PlanId` to know what to replace. `upsert` can take a `Plan` and update by it's (only) unique constraint.